### PR TITLE
[TOKENIZERS] Set cmake project version

### PR DESCRIPTION
**Pull request by mryzhov**
- set default cmake default version (<Openvino Version>.<Extension Version>-<Commit Id>)
- set cpack version 
- disabled cpack source package

### CODE REVIEW

File: modules/custom_operations/CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(openvino_extensions)
+find_package(Git REQUIRED)
```
Comment (ilya-lavrenov) [inline modules/custom_operations/CMakeLists.txt:7]: git can be unavailable, e.g. when we are building on `conda-forge` where archive is used (with no .git folder)

can we move version extraction closer to the place where it's used?

Comment (mryzhov) [reply modules/custom_operations/CMakeLists.txt:7]: good point, will move it closer to cpack configuration

Comment (mryzhov) [reply modules/custom_operations/CMakeLists.txt]: excluded from this PR

File: modules/custom_operations/CMakeLists.txt [hunk 2]
```diff
@@ -4,7 +4,10 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(openvino_extensions)
+set(PROJECT_VERSION 2023.3.0) # aligned with OpenVINO version
+set(PROJECT_VERSION_PATCH 0)
```
Comment (ilya-lavrenov) [inline modules/custom_operations/CMakeLists.txt:8]: looks like we need to use https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION_TWEAK.html#variable:PROJECT_VERSION_TWEAK

File: modules/custom_operations/CMakeLists.txt [hunk 3]
```diff
@@ -13,5 +16,8 @@ find_package(OpenVINODeveloperPackage QUIET)
 if(NOT OpenVINODeveloperPackage_FOUND)
     find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 endif()
+set(CMAKE_PROJECT_VERSION "${OpenVINO_VERSION}.${PROJECT_VERSION_PATCH}")
+
+message("${CMAKE_PROJECT_NAME} version is ${CMAKE_PROJECT_VERSION}")
```
Comment (ilya-lavrenov) [inline modules/custom_operations/CMakeLists.txt:21]: let's remove it, looks like a debug message :)

Comment (mryzhov) [reply modules/custom_operations/CMakeLists.txt]: done

File: modules/custom_operations/CMakeLists.txt (hunk 2 above)
Comment (mryzhov) [reply modules/custom_operations/CMakeLists.txt]: replaced

File: modules/custom_operations/CMakeLists.txt [hunk 4]
```diff
@@ -4,7 +4,10 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(openvino_extensions)
+set(PROJECT_VERSION 2023.3.0) # aligned with OpenVINO version
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply modules/custom_operations/CMakeLists.txt]: changed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**